site/content/docs/latest/reference/testing/ci.md

```diff
@@ -1,105 +1,188 @@
-  - `GKE_REGULAR_VERSION_MAIN` and `GKE_REGULAR_VERSION_LATEST_RELEASE` (on tag or prerelease): the same as above, but using the Kubernetes regular version in GKE.
-  - `sync_chart_to_bitnami` (on tag): when releasing, it will synchronize our development chart with the [bitnami/charts repository](https://github.com/bitnami/charts/tree/main/bitnami/kubeapps) and merge the changes. This step involves:
-    - Checking if the Kubeapps development chart version is greater than the Bitnami chart version. If not, stop.
-    - Deleting the local `bitnami/kubeapps` folder (note that the changes are already committed in git).
-    - Cloning the fork [kubeapps-bot/charts repository](https://github.com/kubeapps-bot/charts/tree/main/bitnami/kubeapps), pulling the latest upstream changes and pushing them back to the fork.
-    - Retrieving the latest version of the chart provided by Kubeapps.
-    - Renaming the development images (`kubeapps/xxx`) by the production ones (`bitnami/kubeapps-xxx`) with the `vX.X.X` tag.
-    - Using `vX.X.X` as the `appVersion`.
-    - Sending a draft PR to the Bitnami Charts repository with these changes (from the robot account's personal fork)
-  - `release` (on tag): it creates a GitHub release based on the current tag by running the script [script/create_release.sh](https://github.com/vmware-tanzu/kubeapps/blob/main/script/create_release.sh).
-
-Note that this process is independent of the release of the official Bitnami images and chart. These Bitnami images will be created according to their internal process (so the Golang, Node or Rust versions we define here are not used by them. Manual coordination is expected here if a major version bump happens to occur).
-
-Also, note it is the Kubeapps team that is responsible for sending a PR to the [chart repository](https://github.com/bitnami/charts/tree/main/bitnami/kubeapps) each time a new chart version is to be released. Even this process is automatic (using the `sync_chart_to_bitnami` workflow), Kubeapps maintainers must manually review the draft PR and convert it into a normal one once it is ready for review.
+# Understanding CI configuration 
+
+Kubeapps leverages GitHub Actions (GHA) for its CI processes: running the tests (both unit and integration tests), building and pushing the images, 
+and syncing the Helm chart with the official [Bitnami chart](https://github.com/bitnami/charts/tree/main/bitnami/kubeapps). 
+The following image depicts how a successful workflow looks like after pushing a commit to the main branch.
+
+![CI workflow after pushing to the main branch](../../img/ci-workflow-main.png "CI workflow after pushing to the main branch")
+
+The different parts involved in the GHA configuration are:
+* **Workflows:** they are what we commonly call `pipelines`. A workflow is a directed acyclic graph composed of several jobs, and it can be automatically
```

Comment:
```suggestion
* **Workflows:** these are what we commonly call `pipelines`. A workflow is a directed acyclic graph composed of several jobs, and it can be automatically
```